bundles/SyliusResourceBundle/configuration.rst

```diff
@@ -1,294 +1,80 @@
-Configuring your resources
+Configuring Your Resources
 ==========================
 
-Now you need to configure your resources! It means that you will tell to this bundle which model, controller, repository, etc.
-will be used for each configured resources. It exists two ways for doing that, we will call them **Basic configuration** and
-*Advanced configuration*. You need to use the first one if you want to build a simple application. It is pretty easy because you just
-need to write configuration (in your config.yml, for example). The second one allows you to embed configuration into your bundles
-but you will need to write some extra lines of code. We will explain the both ways in the next chapters.
+Now you need to configure your first resource. Let's assume you have a *Book* entity in your application and it has simple fields:
 
-Basic configuration
--------------------
+* title
+* author
+* description
 
-In your ``app/config.yml`` (or in an imported configuration file), you need to define which resources you want to use :
+In your ``app/config/config.yml`` add:
 
 .. code-block:: yaml
 
     sylius_resource:
         resources:
-            my_app.entity_name:
+            app.book:
                 driver: doctrine/orm
```

Comment:
Yeah, I guess we can skip that and mention that somewhere else, when we add mongodb docs etc.